### PR TITLE
fix: non flushed io

### PR DESCRIPTION
**Pull request by PrashantRaj18198**
## Desc
Some I/O ops are maintained in buffer, we need to flush them before taking a snapshot. Adding logic in the cleanup script for snapshot to do this.

### CODE REVIEW

File: src/snapshotter/snapshotter.ts [hunk 1]
```diff
@@ -74,6 +74,13 @@ set -e
 # Remove /var/lib/warpbuild-agentd/settings.json
 sudo rm /var/lib/warpbuild-agentd/settings.json
 
+# This command forces a write of all buffered I/O data to the disks. 
+echo "Flushing file system buffers..."
+sync
+    
+# Pause for a few seconds to ensure all I/O operations are complete
+sleep 5
```
Comment (prajjwaldimri) [inline src/snapshotter/snapshotter.ts:82]: Why this arbitrary waiting period? Don't we have some way to check if all the I/O operations were finished?

Comment (PrashantRaj18198) [reply src/snapshotter/snapshotter.ts]: sync automatically does this, this should not be required. Removing this.

Comment (PrashantRaj18198) [reply src/snapshotter/snapshotter.ts]: removed

**Review by prajjwaldimri — APPROVED** [review]
LGTM 🗿

**Pull request merged**